extension/packages/hardhat/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@openzeppelin/contracts": "^4.9.5"
```

Comment:
Good point! When I started to work on this pr openzeppelin version was outdated and I needed to create higher version. But now I can't just use v5+ , since `Counters.sol` was removed from openzeppelin but we need it in our `YourCollectible.sol` code. To fix it, I need to rewrite part of the code. For example, working version with v5+, but I think it's better to change it in another PR so we can test it properly

<details><summary>Code</summary>
<p>

```
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.2; //Do not change the solidity version as it negatively impacts submission grading

import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
import "@openzeppelin/contracts/access/Ownable.sol";

contract YourCollectible is
	ERC721,
	ERC721Enumerable,
	ERC721URIStorage,
	Ownable
{
	uint256 private _nextTokenId;

	constructor() ERC721("YourCollectible", "YCB") Ownable(msg.sender) {}

	function _baseURI() internal pure override returns (string memory) {
		return "https://ipfs.io/ipfs/";
	}

	function mintItem(address to, string memory uri) public returns (uint256) {
		uint256 tokenId = _nextTokenId++;
		_safeMint(to, tokenId);
		_setTokenURI(tokenId, uri);
		return tokenId;
	}

	// The following functions are overrides required by Solidity.

	function _burn(uint256 tokenId) internal override(ERC721) {
		super._burn(tokenId);
	}

	function tokenURI(
		uint256 tokenId
	) public view override(ERC721, ERC721URIStorage) returns (string memory) {
		return super.tokenURI(tokenId);
	}

	function _increaseBalance(
		address account,
		uint128 value
	) internal virtual override(ERC721, ERC721Enumerable) {
		super._increaseBalance(account, value);
	}

	function _update(
		address to,
		uint256 tokenId,
		address auth
	) internal virtual override(ERC721, ERC721Enumerable) returns (address) {
		return super._update(to, tokenId, auth);
	}

	function supportsInterface(
		bytes4 interfaceId
	)
		public
		view
		override(ERC721, ERC721Enumerable, ERC721URIStorage)
		returns (bool)
	{
		return super.supportsInterface(interfaceId);
	}
}

```

</p>
</details>